src/Model.php

```diff
@@ -352,7 +352,7 @@ public function validate($intent = null)
      */
     public function addField($name, $defaults = [])
     {
-        $field = $this->factory($this->mergeSeeds($this->_default_seed_addField, $defaults), null, 'Field');
+        $field = $this->factory($this->mergeSeeds($defaults, $this->_default_seed_addField), null, '\atk4\data\Field');
```

Comment:
which one will take precedence here - `$defaults` or `$this->_default_seed_addField` ?

Reply:
left has precedence.